pkgs/tools/misc/coreutils/default.nix

```diff
@@ -11,7 +11,7 @@ assert selinuxSupport -> libselinux != null && libsepol != null;
 with { inherit (stdenv.lib) optional optionals optionalString optionalAttrs; };
 
 let
-  self = stdenv.mkDerivation rec {
+  self = rec {
```

Comment:
Now self turns into a plain attribute set instead of derivation. I believe that won't do good with the line
```bash
cp ${self}/share/man/man1/* $out/share/man/man1
```
(present in cross-compilation only)